keanu-project/src/test/java/io/improbable/keanu/algorithms/mcmc/NUTSTest.java

```diff
@@ -53,18 +97,19 @@ public void samplesContinuousPrior() {
 
         BayesianNetwork bayesNet = MCMCTestDistributions.createSumOfGaussianDistribution(20.0, 1.0, 46., 15.0);
 
-        int sampleCount = 6000;
+        int sampleCount = 5000;
         NUTS nuts = NUTS.builder()
             .adaptCount(sampleCount)
-            .maxTreeHeight(4)
+            .maxTreeHeight(10)
```

Comment:
This is now very slow... but we didn't want to mark this test as `@Slow` because it's the only non-slow one we have, that we run every time.

Reply:
Reduced the number of samples & started it a little closer to the right answer